File: java/src/com/thoughtworks/cep/conference/FeatureToggles.java
package com.thoughtworks.cep.conference;

import java.util.ArrayList;
import java.util.List;

import static java.util.Objects.requireNonNullElseGet;

public class FeatureToggles {
    private final List<String> enabledFeature;

    public FeatureToggles(List<String> enabledFeatures) {
        this.enabledFeature = requireNonNullElseGet(enabledFeatures, ArrayList::new);
    }

    public boolean isEnabled(String feature) {
        return this.enabledFeature.contains(feature);
    }
}

File: csharp/CepConference/FeatureToggles.cs
namespace CepConference;

public class FeatureToggles
{
    private readonly ICollection<string> _enabledFeatures;

    public FeatureToggles(ICollection<string> enabledFeatures)
    {
        _enabledFeatures = enabledFeatures;
    }

    public bool IsEnabled(string feature)
    {
        return _enabledFeatures.Contains(feature);
    }
}